dev/README_RELEASE_PYTHON_CLIENT.md

```diff
@@ -132,7 +132,7 @@ git log 2.8.0..HEAD --pretty=oneline -- airflow-core/src/airflow/api_fastapi/cor
 ```shell script
 cd ${AIRFLOW_REPO_ROOT}
 rm dist/*
-breeze release-management prepare-python-client --distribution-format both --python-client-repo "${CLIENT_REPO_ROOT}"
+breeze release-management prepare-python-client --distribution-format both --python-client-repo "${CLIENT_REPO_ROOT}" --version-suffix ""
```

Comment:
No suffix? Not rc1 or rc2?

Reply:
Ah.. The problem is that we already set `VERSION_SUFFIX` as variable above and `--version-suffix` takes value from the variable. Yeah. Lilkely good idea to override it with ""

@jscheffl -> yes, when we prepare artifacts for `SVN` we "pretend" they are final and put them in `${VERSION_RC}" folder in SVN. This is important because when this rc becomes FINAL, we can just move it over to the "release" SVN tree - so that it's THE SAME file - binary identical - as we voted on. Otherwise we would have to again rebuild it without rc.